src/app/api/digests/es6/digestsFormatAsHal.js

```diff
@@ -0,0 +1,34 @@
+export default (href, instanceId, digests) => {
+const response = {
+    "_links": {
+      "self": {
+        "href": href(`/api/${instanceId}/digests`)
+      }
+    },
+    "count": digests ? digests.length : 0,
+    "_embedded": {
+      "digests": []
+    }
+  }
+
+  function createDigestHyperMediaResult(digest) {
```

Comment:
Can replace this with `const create...Result = digest => { ` arrow function syntax